warehouse/templates/manage/account.html

```diff
@@ -219,7 +219,7 @@ <h2 class="sub-title">Change Password</h2>
     </div>
     <div class="form-group">
       <label class="form-group__label" for="name">New Password</label>
-      {{ change_password_form.new_password(placeholder="Select a new password", required="required", class_="form-group__input") }}
+      {{ add_password_strength(change_password_form.new_password(placeholder="Select a new password", required="required", class_="form-group__input")) }}
```

Comment:
Is there a reason why we need to pass the field into the macro like this? I was imagining this being something like:

```
<label class="form-group__label" for="name">New Password</label>
{{ change_password_form.new_password(placeholder="Select a new password", required="required", class_="form-group__input") }}
{{ field_errors(change_password_form.new_password) }}
{{ password_strength_gauge() }}
```
(Also I think the errors should probably come before the gauge)

Reply:
The input field needs to be wrapped in a `div` tag to properly place the tooltips since they're implemented on pseudo-elements. Since it's not immediately obvious I was trying to avoid having to manually do it on each template but it just makes things less explicit, fixed now.